adventure_game/main.py

```diff
@@ -31,7 +33,7 @@ def item_pickup():
     print_pause("You see a faint glimmer in the sunshine")
     print_pause("You notice that it is in fact a sword")
     choice = valid_input("Do you want to pick up the sword? Y/N ", "Y", "N")
-    if choice == "Y":
+    if choice in Y:
```

Comment:
really you want to create a function similar to the valid_input one, to avoid redundancy. Here you're checking twice which is not bad but you want to make you life easier. Check out [Ellen's input file for an example](https://github.com/arcadecoder/Learning_collab/blob/master/adventure_game/Ellen_input.py)